.babelrc

```diff
@@ -35,6 +35,12 @@
       "plugins": [
         "./build/use-lodash-es"
       ]
+    },
+    "rollup": {
+      "plugins": [
+        "./build/use-lodash-es",
+        "external-helpers"
```

Comment:
I started out using Rollup in a widget-like app recently, so I've run into this before. Ended up just saying screw IE for now, since supporting any browser with Promise support was a big amount of bloat (double the size, roughly).

core-js is a polyfill, though. That's all it does. As long as you have `rollup-plugin-commonjs` set up (you do), it will just work. 

However, if you externalize the helpers, it's going to be a hidden dependency. And if you don't transform-runtime them, rollup-plugin-babel is probably going to complain as well. I assume that's why you added this in the first place?